DIsability VA Nepal.R

```diff
@@ -1,12 +1,25 @@
 #import data from an R project I have titled "Nepal disability and VA"
 library(tidyverse)
-NEW2VIEWIIPilotFinal_DATA_2019.09.03_1817 <- read.csv("~/Desktop/Gap Year/Gap Year Projects/Nepal VA and Disability/NEW2VIEWIIPilotFinal_DATA_2019-09-03_1817.csv", row.names=1)
+# JK: Not sure of all the reasons why it's better but I use readr and its command read_csv not read.csv
+library(readr)
+# JK: Also, if I didn't mention it before, you need to create a project file for each project.
+# JK: go to File tab in RStudio and "New Project" and pick/create a directory
+# JK: Then each time you want to work in R, first open the project, and then from within the project open the R script
+# JK: Multiple benefits to this system. One of them is that you don't need to ever specify the directory, which makes it easier when we share files
+# JK: But other benefits too, including that you always start with a clean slate so it makes everything more reproducible.
+# JK: if you don't start with a clean slate you might have packages open from a prior analysis, and then these can interfere with what you are trying to do
+# JK: Once you have a project you should keep all the csvs and R scripts for that project in the same folder/directory
```

Comment:
I started creating projects after your feedback on my Alto amazonas SES code. I state the name of the project in line 1, is there a better way to indicate it?